var/spack/repos/builtin/packages/symengine/package.py

```diff
@@ -35,7 +36,9 @@ class Symengine(Package):
     version('0.1.0', '41ad7daed61fc5a77c285eb6c7303425')
     version('develop', git='https://github.com/symengine/symengine.git')
 
-    variant('flint',        default=True,
+    variant('boost',        default=True,
```

Comment:
If this is by default `True`, then `mpc` and `mpfr` should be false by default. Since spack gives you `gmp, mpfr and mpc`, you can have them by default since `gmp` is faster than `boostmp` unless you want a build without LGPL. (boost=True would disable arbitrary-precision evaluations as well.)
Also, how about changing the option to `boostmp` instead of `boost`?

Also,